gauss_plot_2d.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p>Add meaningful title in first cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>